custom_components/tuya_local/devices/moebot_s_mower.yaml

```diff
@@ -297,3 +297,11 @@ entities:
         type: boolean
         name: sensor
         optional: true
+  - entity: switch
+    name: Hedgehog protection
+    icon: "mdi:account-group"
+    category: config
+    dps:
+      - id: 118
+        type: boolean
```

Comment:
You will need to use `optional: true` here, as this seems to be a new feature that is not on earlier models.

Reply:
you are right, I've missed that
added optional: true.